src/patterns/molecules/sticky-promo/sticky-promo.hbs

```diff
@@ -20,6 +22,11 @@ notes: |
   <a class="mzp-c-button mzp-t-product mzp-t-small" href="#">Download Now</a>
 </aside>
 
-<script
- src="../../../assets/protocol/protocol/js/protocol-sticky-promo.js">
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-sticky-promo.js"></script>
+
+<script>
+(function() {
+    'use strict';
+    Mzp.StickyPromo.open();
```

Comment:
When I make a button to show the promo onclick, then close it, then click to show it again, it remains invisible because it still has the `mzp-a-fade-out` class. I think the close event should reset it back to its original state.